docs/learn/learn-async-backing.md

```diff
@@ -7,103 +7,232 @@ keywords: [parachains, backing, parablock, perspective parachains, unincluded se
 slug: ../learn-async-backing
 ---
 
-Asynchronous backing is a feature that introduces
-[pipelining](https://www.techtarget.com/whatis/definition/pipelining) to the parachain block
-generation and validation process. It is analogous to the logical pipelining of processor
-instruction in "traditional" architectures, where some instructions may be executed before others
-are complete. Instructions may also be executed in parallel, enabling multiple parts of the
-processor to be working on potentially different instructions at the same time.
+import RPC from "./../../components/RPC-Connection";
 
-Bundles of state transitions represented as blocks may be processed similarly. In the context of
-Polkadot, it aims to increase the throughput of the entire network.
+:::info Learn about Parachain Consensus
+
+To fully follow the material on this page, it is recommended to be familiar with the primary stages
+of the [Parachain Protocol](./learn-parachains-protocol.md).
+
+:::
+
+## Synchronous Backing
+
+Before diving into asynchronous backing, it is important to understand what synchronous backing is
+and what its main limitations are.
+
+:::info What is backing?
+
+**Backing** refers to the process in which a parablock is verified by a subset of validators who
+determine the block’s validity. It is an important step in the validation process for parablocks, as
+it is the first line of defense in ensuring valid state transitions. Validators who back the
+parablock put their stake at risk, and if the block is later found invalid, those validators are
+slashed.
+
+:::
+
+In synchronous backing, parablock generation is tightly coupled to the relay chain's progression on
+a one-to-one basis. Every parablock must be generated and backed within a relay-chain block
+(six-second window), and (if successfully backed) it will be included in a relay-chain block (often
+referred to as the **parent** block, as the parablock anchors itself to it) after an additional six
+seconds. Thus, a parablock can be produced every 12 seconds because a new parablock can be produced
+after including the previous one.
+
+The **contextual execution** of new parablocks are parablock ancestors included in the relay chain
```

Comment:
I don't see why we're discussing contextual execution at this point. I'd suggest either removing this paragraph entirely or rewording it and placing it later on. 

Put simply, here's the difference in execution context between synchronous and asynchronous backing:

Synchronous - The entire execution context comes from the latest relay block and most recently included parablock from the same parachain.

Asynchronous - A baseline execution context is first set up using a parablock's relay parent and the most recently included parablock as of that relay parent. The relay parent need not be the most recent relay block.

From here, we modify the context for each unincluded parablock predecessor of the block under construction. Said predecessors live in the unincluded segment.

In this way, synchronous backing uses execution context entirely pulled from the relay chain. While asynchronous backing augments this with off-chain context from the unincluded segment.